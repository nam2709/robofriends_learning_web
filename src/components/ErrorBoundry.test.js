import React from 'react';
import { render } from '@testing-library/react';
import ErrorBoundry from './ErrorBoundry';

// Ensure that Jest knows how to handle .snap files
import '@testing-library/jest-dom/extend-expect';

test('MyComponent snapshot', () => {
  const { asFragment } = render(<ErrorBoundry />);
  expect(asFragment()).toMatchSnapshot();
});

// test('correctly error', () => {

// })