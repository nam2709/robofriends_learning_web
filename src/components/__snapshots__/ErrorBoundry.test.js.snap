// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent snapshot 1`] = `<DocumentFragment />`;
